### PR TITLE
Creates a placeholder for the npm-in-a-box scifi exhibit

### PR DESCRIPTION
**Pull request by momack2**
Alex - can you help fill this out?

### CODE REVIEW

File: SCI-FI_FAIR/presenter-npm-in-a-box.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# What is the NAME of your project?
+NPM in a box
+
+# What DOES THE PROJECT DO/what will you SHOW?
+_Briefly explain your madcap decentralized science experiment and why it will excite and inspire the fair attendees :)_
```
Comment (achingbrain) [inline SCI-FI_FAIR/presenter-npm-in-a-box.md]:
```suggestion
npm in a box is your own personal mirror for the public npm registry running on a Raspberry Pi connected to your local network.

Control your own little corner of the JavaScript Commons and install packages from it using [`ipfs-npm`](https://www.npmjs.com/package/ipfs-npm)

Also use it to publish [IPNS](https://docs.ipfs.io/guides/concepts/ipns/) names for your own modules published on IPFS!
```

File: SCI-FI_FAIR/presenter-npm-in-a-box.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# WHO will PRESENT the project?
+_PREFERRED NAME(S) and CONTACT EMAIL(S), please. The fair takes place 16:30 - 18:30 on Saturday, booked against the Unconference, so we recommend assigning 2-3 people in shifts, if possible. If you'll need to leave your post unattended for part of the time, please note that._
```
Comment (achingbrain) [inline SCI-FI_FAIR/presenter-npm-in-a-box.md]:
```suggestion
Alex Potsides - alex.potsides@protocol.ai
```

File: SCI-FI_FAIR/presenter-npm-in-a-box.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+NPM in a box
+
+# What DOES THE PROJECT DO/what will you SHOW?
+_Briefly explain your madcap decentralized science experiment and why it will excite and inspire the fair attendees :)_
+
+# WHO will PRESENT the project?
+_PREFERRED NAME(S) and CONTACT EMAIL(S), please. The fair takes place 16:30 - 18:30 on Saturday, booked against the Unconference, so we recommend assigning 2-3 people in shifts, if possible. If you'll need to leave your post unattended for part of the time, please note that._ 
+
+# WHAT EQUIPMENT will you need?
+_The standard equipment is a table with a power strip. We can provide a MONITOR if needed (specify it now so we can rent one). OTHER EQUIPMENT or materials can also be provided within reason -- we want this fair to be distinctly more fun than the average conference poster session so creative execution ideas are welcome! Please include any ASSISTIVE DEVICES needed._
```
Comment (achingbrain) [inline SCI-FI_FAIR/presenter-npm-in-a-box.md]:
```suggestion
Just a table & somewhere to plug the pi in.
```

File: SCI-FI_FAIR/presenter-npm-in-a-box.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+# Do you have a LOGO/HERO IMAGE for the main projector?
+_Please link an image you'd like shown in the slideshow on the main projector in this Fair room (1280 x 720 or higher resolution image or 720p video loop/gif of <10sec)_
```
Comment (achingbrain) [inline SCI-FI_FAIR/presenter-npm-in-a-box.md]: I like this one:

jen-npm

But this one is probably better:

npm-in-a-box

So, uh:

```suggestion
![npm-in-a-box](https://user-images.githubusercontent.com/665810/60212987-d5a3a780-9862-11e9-87c5-88c419f50411.png)
```

File: SCI-FI_FAIR/presenter-npm-in-a-box.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# What is the NAME of your project?
+NPM in a box
```
Comment (achingbrain) [inline SCI-FI_FAIR/presenter-npm-in-a-box.md]:
```suggestion
npm in a box
```

Comment (achingbrain) [reply SCI-FI_FAIR/presenter-npm-in-a-box.md]: (they spell it lower case..)

**Comment by achingbrain** [comment]
TBH there's going to be a fair bit of hand-waving as IPFS is setting fire to the prototype, it's just too heavy for a Raspberry Pi at the moment..

**Review by daviddias — APPROVED** [review]

**Pull request merged**